grape.gemspec

```diff
@@ -11,6 +11,12 @@ Gem::Specification.new do |s|
   s.summary     = 'A simple Ruby framework for building REST-like APIs.'
   s.description = 'A Ruby framework for rapid API development with great conventions.'
   s.license     = 'MIT'
+  s.metadata    = {
+    'bug_tracker_uri'   => 'https://github.com/ruby-grape/grape/issues',
```

Comment:
Do these have to be strings? or `bug_tracker_uri: 'http...'` which would read a bit cleaner. I am totally nitpicking.